### DISCUSSION TITLE
July 18th 2024 PowerShell/OpenSSH Community Call

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PowerShell#23977] [Community Call]
Agenda 

- Community PR update

**Comment by StevenBucher98** [comment]
- Answer some FAQs from PSConfEU

**Comment by ThomasNieto** [comment]
Microsoft.PowerShell.Archive repository archived: https://github.com/PowerShell/PowerShell/discussions/24006

**Comment by iSazonov** [comment]
> **Announcement**: [LocalAccounts](https://github.com/iSazonov/LocalAccounts) module has been published.
> It is a port of Windows PowerShell `Microsoft.PowerShell.LocalAccounts` module to .Net 8.0+ for using with PowerShell 7.4+ based on [LocalAccounts-MS](https://github.com/iSazonov/LocalAccounts-MS).
> While `LocalAccounts-MS` is a port of the original `Microsoft.PowerShell.LocalAccounts` module for maintaining maximum backward compatibility and it is frozen, the `LocalAccounts` project is open for development.

To install:
```powershell
Install-Module -Name LocalAccounts -AllowPrerelease
```

**Comment by pbossman** [comment]
Demo:
Get 5 Min to Discuss PowerShell Saturday NC.
- What is it?
- Who is it for?
- How to attend?
- How to participate?
